easybuild/framework/easyblock.py

```diff
@@ -2156,7 +2156,7 @@ def extensions_step(self, fetch=False):
             # always go back to original work dir to avoid running stuff from a dir that no longer exists
             change_dir(self.orig_workdir)
 
-            tup = (ext.name, ext.version, idx+1, exts_cnt)
+            tup = (ext.name, ext.version or '', idx+1, exts_cnt)
```

Comment:
Can there be a version of `None` at this point? How? Other uses of `.version`: `topdir = os.path.join(self.builddir, '%s-%s' % (self.name, self.version))` so this looks like omitting the `or...` is save

Reply:
We have situations where extensions are specified only by name, which just serves as a check that these extensions are installed (and then the version is indeed `None`).

Try removing the `or ''`, and run the test with `python -O -m test.framework.easyblock skip_ext`

Reply:
Ah, ok. R seems to have some of those.